.gitignore

```diff
@@ -1,3 +1,4 @@
 build/
+.idea/
```

Comment:
Please ignore the .idea folder

Reply:
That's exactly what this line means, because it is in the `.gitignore` file.